Contribute case studies to the guide (#1022)
BitcoinDesign/Guide: guide/contribute/case-studies.md

---
layout: guide

> title: Case Studies

Reviewer: ```suggestion
title: Case studies
```
We use sentence case everywhere.

> description: How to contribute case studies to the guide

Reviewer: ```suggestion
description: How to contribute case studies to the guide.
```

nav_order: 10
parent: Contribute to guide
permalink: /guide/contribute/case-studies/
main_classes: -no-top-padding

> image: https://bitcoin.design/assets/images/guide/contribute/contribute-preview.jpg

Reviewer: You need a custom preview image for this page that's 1200x630px.

Reviewer: Still need this image.

---

{% include picture.html

>    image = "assets/images/guide/contribute/case-studies/CasestudiesHeader.png"

Reviewer: [Guidelines for file names](https://bitcoin.design/guide/contribute/illustration-guidelines/#file-names).

JPEG should be smaller here than PNG ([guidelines](https://bitcoin.design/guide/contribute/illustration-guidelines/#jpg-png-or-gif)).

>    retina = "assets/images/guide/contribute/case-studies/CaseStudiesHeader2X.png"

Reviewer: Use `@2x` to indicate retina images.

   mobile = "assets/images/guide/contribute/case-studies/CaseStudiesHeaderMobile.png"
   mobileRetina = "assets/images/guide/contribute/case-studies/CasestudiesHeaderMobile2X.png"

>    alt-text = "Image of people reading books"

Reviewer: The screen reader already knows this is an image, no need to repeat (see [alt text writing tips](https://webaim.org/techniques/alttext/)).

   width = 1600

>    height = 800

Reviewer: ```suggestion
   height = 268
```
Your image file is not 800px tall, but 268px.

   layout = "full-width"
%}

> # Case Study Guidelines

Reviewer: ```suggestion
# Case study guidelines
```

> There are various types of case studies. Some document proven solutions in the ecosystem, others are short, to the point stories of how a designer or team solved a particular challenge. This document is intended to help you write one of the latter. Ideally a case study that is digestible for a wider audience would be in the range of 1-4 pages in length.

Reviewer: ```suggestion
There are various types of case studies. Some document proven solutions in the ecosystem, others are short, to the point stories of how a designer or team solved a particular challenge. This document is intended to help you write one of the latter. Ideally, a case study that is digestible for a wider audience, would be in the range of 1-4 pages in length.
```

Describing the final result is as important as laying out the process of how you got there. That makes a story format ideal for case studies. It keeps readers engaged and allows them to also learn how they can get good results in their own work. A good case study has a nice mixture of visuals and copy that tell a story about how you solved a problem. Also keep in mind when writing that readers are usually scanning the content for key insights.

Most people will visit a case study to:
* Learn how a particular problem was solved
* Bring back or justify an idea about a particular design they would like to include in their own project
* Understand how improvements were implemented
* Understand the bigger insights learnt
* Learn about the steps taken in the process

Below, we will roughly outline steps you can include in your case study. This is by no means a fixed template, but merely ideas which can be put together to share your story. Feel free to change and add in additional information of steps that you feel are important to include.

## How it started
Start by sharing how this particular project/effort started.

### Content ideas
* What problem were you trying to solve?

> * How did this problem come to the source? What was the background?

Reviewer: ```suggestion
* How did this problem come to the source? What was the background?
```
To the `source`? Do you mean `surface`?

Author: Correct, changed

* How did the team collaborate (or perhaps this was a solo project)?
* Share some background of the product itself

### Visuals
* Show your final design solution.

## User base
Most applications have a specific type of user or use case they are optimized for. Try to give the reader some insight as to who this is, and how the application services them. Perhaps the team did not have knowledge about their users at the beginning and did research to try to understand this.

### Visuals
* A graphic showing the user persona the application is targeting
* A list of questions the team used to come up with the user persona

> Example of user persona Blixt wallet case study

Reviewer: ```suggestion
Here's an example of a user persona from the [Blixt wallet case study]({{ '/guide/case-studies/blixt-wallet/' | relative_url }}).
```

{% include picture.html
   image = "assets/images/guide/contribute/case-studies/UserPersonasExampleBlixt.png"
   retina = "assets/images/guide/contribute/case-studies/UserPersonasExampleBlixtX2.png"
   mobile = "assets/images/guide/contribute/case-studies/UserPersonasExampleBlixtMobile.png"
   mobileRetina = "assets/images/guide/contribute/case-studies/UserPersonasExampleBlixtMobileX2.png"

>    alt-text = "User personas example"

Reviewer: The alt text can be more descriptive.

Author: Added.

   width = 1600
   height = 800
   modalWidthMobile = 400

>    modalHeightMobile = 400

Reviewer: ```suggestion
   modalHeightMobile = 271
```
Your mobile image is 271px tall.

%}

> Example of user persona Payjoin case study

Reviewer: Accidental repeat of this sentence? It's also above the image.

## Methodology
What steps led to the final solution? While the specific steps don't need to be detailed extensively, it's important to highlight the ones that provided the most valuable insights. Feel free to mention any challenges faced, key decision points regarding the chosen steps or tools.

### Visuals

> * Images of userflows or snapshots of research done. Anything visual can go here that helps to tell a bit of a story. These can be messy.

Reviewer: ```suggestion
* Images of user flows or snapshots of research done. Anything visual can go here that helps to tell a bit of a story. These can be messy.
```

> * An image showing the main insights, if you have one.

Reviewer: ```suggestion
* An image showing the main insights, if you have them.
```

## Insights
After completing all the steps to solve the particular design challenge, you likely discovered some key insights. Share them here.

### Content ideas
* A blurb summarising what was learnt.

> * Bullet points highlighting the main insights.

Reviewer: Both paragraphs above already talk about main insights. Feels a bit repetitive. Can we mix this up a little? I think it's fine to leave the points about insights, but each section can take a different angle. Visuals can be about annotated screenshots or other material. The Insights paragraph itself can describe those insights in detail. And content ideas can build on those insights to generate improvements.

## Conclusion
This is a final wrap-up where we close off the story. Share what happened after the case study/research was completed.

> ### Content ideas

Reviewer: Don't think you need this "Content ideas" title again. We just saw another one of those just a little further up. Maybe just remove it?

* What was the impact of the solution? Add in qualitative and quantitative data if you have it.
* Was the solution implemented by the project?
* What were the next steps after completing this process?
* Make sure to thank everyone who participated or helped

## Resources

> This is the last section of the case study where it’s nice to share links to all the parts; or rather the puzzle pieces. Here you can add in simple titles such as a Figma file or links to research conducted. Add all the artefacts that were used in the entire design process to give the reader the option to dive in and keep learning from your experience.

Reviewer: ```suggestion
This is the last section of the case study where it’s nice to share links to all the parts; or rather the puzzle pieces. Here you can add in simple titles such as a design file or links to research conducted. Add all the artefacts that were used in the entire design process to give the reader the option to dive in and keep learning from your experience.
```

## Acknowledgements/Gratitude
It is a good idea to acknowledge people, projects or organisations that provided resources, input, feedback etc. for the case study.

Related resources:

> The below 2 case studies were put together roughly using the above guidelines.

Reviewer: ```suggestion
The below two case studies were put together using roughly the above guidelines.
```

> * [Blixt](/guide/case-studies/blixt-wallet/)

Reviewer: ```suggestion
* [Blixt]({{ '/guide/case-studies/blixt-wallet/' | relative_url }})
```
We need to use this more complicated format for internal links.

> * [The Payjoin experience](/guide/case-studies/payjoin/)

Reviewer: ```suggestion
* [The Payjoin experience]({{ '/guide/case-studies/payjoin/' | relative_url }})
```

---

{% include next-previous.html

>    previousUrl = "/guide/review/"

Reviewer: Previous page is not right, should be the [Media page](https://bitcoin.design/guide/contribute/formatting/media/).

   previousName = "Review"

>    nextUrl = "/guide/daily-spending-wallet/"

Reviewer: Next page is not right. Should be the [Glossary](https://bitcoin.design/guide/glossary/).

---
layout: guide
title: Case studies
description: How to contribute case studies to the guide.
nav_order: 10
parent: Contribute to guide
permalink: /guide/contribute/case-studies/
main_classes: -no-top-padding
image: https://bitcoin.design/assets/images/guide/contribute/contribute-preview.jpg
---

{% include picture.html

>    image = "assets/images/guide/contribute/casestudies-header.jpg"

Reviewer: All the file paths are incorrect here. Please double check.

   retina = "assets/images/guide/contribute/contribute/casestudies-header@2X.jpg"
   mobile = "assets/images/guide/contribute/casestudies-header-mobile.jpg"
   mobileRetina = "assets/images/guide/contribute/casestudies-header-mobile@2X.jpg"

>    customImage = "assets/images/guide/contribute/casestudy-contribute-custom-image.jpg

Reviewer: Not sure where this `customImage` attribute comes from, but this does not exist in `picture.html`. You can remove it.

   alt-text = "people reading books"
   width = 1600

>    height = 268

Reviewer: ```suggestion
   height = 264
```

## How it started
Start by sharing how this particular project/effort started.

### Content ideas
* What problem were you trying to solve?
* How did this problem come to the surface? What was the background?
* How did the team collaborate (or perhaps this was a solo project)?
* Share some background of the product itself

### Visuals
* Show your final design solution.

## User base
Most applications have a specific type of user or use case they are optimized for. Try to give the reader some insight as to who this is, and how the application services them. Perhaps the team did not have knowledge about their users at the beginning and did research to try to understand this.

### Visuals
* A graphic showing the user persona the application is targeting
* A list of questions the team used to come up with the user persona

Here's an example of a user persona from the [Blixt wallet case study]({{ '/guide/case-studies/blixt-wallet/' | relative_url }}).
{% include picture.html

>    image = "assets/images/guide/contribute/case-studies/user-persona-example-blixt.jpg"

Reviewer: Image is not rendering due to inconsistencies in the file names.
